dChatServer/ChatPacketHandler.cpp

```diff
@@ -354,6 +354,75 @@ void ChatPacketHandler::HandleGMLevelUpdate(Packet* packet) {
 	inStream.Read(player.gmLevel);
 }
 
+
+void ChatPacketHandler::HandleWho(Packet* packet) {
+	CINSTREAM_SKIP_HEADER;
+	LWOOBJID playerID;
+	inStream.Read(playerID);
+
+	const auto& sender = Game::playerContainer.GetPlayerData(playerID);
+	if (!sender) return;
+
+	LUWString playerName;
+	inStream.Read(playerName);
+	const auto& player = Game::playerContainer.GetPlayerData(playerName.GetAsString());
+	bool online = true;
+	if (!player) online = false;
+
+	CBITSTREAM;
+	BitStreamUtils::WriteHeader(bitStream, eConnectionType::CHAT, eChatMessageType::WORLD_ROUTE_PACKET);
+	bitStream.Write(playerID);
+
+	BitStreamUtils::WriteHeader(bitStream, eConnectionType::CLIENT, eClientMessageType::WHO_RESPONSE);
+	bitStream.Write<uint8_t>(online);
+	bitStream.Write(player.zoneID.GetMapID());
+	bitStream.Write(player.zoneID.GetInstanceID());
+	bitStream.Write(player.zoneID.GetCloneID());
+	bitStream.Write(playerName);
+
+	SystemAddress sysAddr = sender.sysAddr;
+	SEND_PACKET;
+}
+
+void ChatPacketHandler::HandleShowAll(Packet* packet) {
+	CINSTREAM_SKIP_HEADER;
+	LWOOBJID playerID;
+	inStream.Read(playerID);
+
+	const auto& sender = Game::playerContainer.GetPlayerData(playerID);
+	if (!sender) return;
+
+	LUString displayZoneData(1);
+	inStream.Read(displayZoneData);
+	LUString displayIndividualPlayers(1);
+	inStream.Read(displayIndividualPlayers);
```

Comment:
a 1 character LUString?  just read a character in that case?

Reply:
this was stupid, I just need to read the text in the slash command and write 0 or 1 instead of passing the text along internally and then doing the comparison here, will fix